### DISCUSSION TITLE
How are release notes generated?

### DISCUSSION THREAD

**Discussion by kristojorg** [vercel/swr#1654] [Q&A] [answered]
Hi there,

How are the release notes for this repo generated on github? I would like to do something similar for a project, but can't seem to find a dependency or config somewhere that is generating them. Do you mind pointing me to the right place to look? Thank you!

**Comment by huozhi** [comment, marked as answer]
Hi, please try [vercel/release](https://github.com/vercel/release) !

**Reply by kristojorg** [reply]
Thanks!
